lib/nostrum/consumer.ex

```diff
@@ -123,16 +136,24 @@ defmodule Nostrum.Consumer do
   @type message_reaction_remove :: {:MESSAGE_REACTION_REMOVE, {map}, WSState.t()}
   @type message_reaction_remove_all :: {:MESSAGE_REACTION_REMOVE_ALL, {map}, WSState.t()}
   @type message_ack :: {:MESSAGE_ACK, {map}, WSState.t()}
+  @typedoc """
+  Dispatched when a user's presence is updated.
+  `old_presence` will be `nil` when the pre-update presence could not be fetched from the cache.
```

Comment:
Could you add a new line between the two sentences? ExDoc uses the first line to generate summaries, so we should try to keep it away from the rest of the provided info just for clarity.